Library/Homebrew/language/python.rb

```diff
@@ -88,6 +88,17 @@ def self.setup_install_args(prefix)
       ]
     end
 
+    def self.rewrite_python_shebang(python_path)
+      Pathname(".").find do |f|
+        regex = %r{^#! ?/usr/bin/(env )?python([23](\.\d+)?)$}
```

Comment:
```suggestion
        regex = %r{^#! ?/usr/bin/(env )?python([23](\.\d{1,2})?)$}
```

Last suggested change just for maximum pedantry so `maximum_regex_length` is definitely correct (if we end up with `3.100` it'll otherwise match the regex and fail the read).